ibm/config.go

```diff
@@ -2738,6 +2752,39 @@ func (c *Config) ClientSession() (interface{}, error) {
 			"X-Original-User-Agent": {fmt.Sprintf("terraform-provider-ibm/%s", version.Version)},
 		})
 	}
+	
```

Comment:
Please add below line [after](https://github.com/IBM-Cloud/terraform-provider-ibm/blob/a0d7985697370c22c1991ae74d4ad8e9c66517fa/ibm/config.go#L1130) 
`session.postureManagementClientErrv2 = errEmptyBluemixCredentials`

Could you add this line for v1 version as well..